cmake.config.h.in

```diff
@@ -0,0 +1,73 @@
+/* config.h.in.  Generated from configure.ac by autoheader.  */
```

Comment:
If it is autogenerated, does it have to be included?

Reply:
I overlooked updating this comment when I translated the generated file to cmake format. Updated.

Reply:
OK, no problem!